components/movementsensor/adxl345/adxl345.go

```diff
@@ -80,10 +123,27 @@ func init() {
 		&AttrConfig{})
 }
 
+type interruptInfo struct {
```

Comment:
This and incrementCount are unnecessary IMO. board.DigitalInterrupt already has a Value() method.

Reply:
This counts how many times the specific interrupt has been seen (single tap or free fall) instead of the number of interrupts that have gone off on the pin